collections/ansible_collections/cloudkit/service/roles/manage_agents/tasks/remove_agents.yaml

```diff
@@ -0,0 +1,57 @@
+- name: Display node names that will be removed from being agents
+  ansible.builtin.debug:
+    var: manage_agents_node_names
+
+- name: Get list of existing agents
+  kubernetes.core.k8s_info:
+    kind: Agent
+    api_version: agent-install.openshift.io/v1beta1
+    namespace: "{{ manage_agents_namespace }}"
+  register: initial_agents
```

Comment:
I don't think `initial_agents` is used, is it? Regardless, `get_agent_names` could remove its own task fetching agents, and just have this value passed in.